src/damage.cpp

```diff
@@ -190,7 +190,11 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
             g->m.add_field( pt, fd_tear_gas, 3, 0 );
         }
     }
-
+    if( effects.count( "GAS_FUNGICIDAL" ) > 0 ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
+            g->m.add_field( pt, fd_fungicidal_gas, 4, 0 );
```

Comment:
`map::add_field` clips the density to 3, so having 4 here won't do anything.

Not sure what you mean by 'sinking in' here. That would probably be achieved better by changing the numbers in field processing, though. `spread_gas` has arguments like spread chance, outdoor dispersal speedup etc.